### PR TITLE
Added Dark Theme

### PR DESCRIPTION
**Pull request by Nishant8209**
Added dark theme ,added quick modal in the plp page 
dark
Screenshot 2024-11-08 145115

### CODE REVIEW

File: src/libs/ui/assets/Arrow-down-black.63216089.svg
Comment (Vishnu9901) [inline src/libs/ui/assets/Arrow-down-black.63216089.svg:1]: Where your using this SVG

File: src/libs/ui/src/molecules/PopularProductHeading/index.tsx [hunk 1]
```diff
@@ -17,13 +17,13 @@ const ProductHeader = ({
   return (
     <div className="text-left  ">
       <h2
-        className={`font-bold py-2 mt-20 text-[28px] text-[#555555] font-HeroNewExtraBold leading-8 hover:font-HeroNewBold ${className}`}
+        className={`font-bold py-2 mt-20 text-[28px] text-[#555555] font-HeroNewExtraBold leading-8 hover:font-HeroNewBold dark:text-appTextColor${className}`}
       >
```
Comment (Vishnu9901) [inline src/libs/ui/src/molecules/PopularProductHeading/index.tsx:21]: give some space between classes
 dark:text-appTextColor${className}

File: src/libs/ui/src/organisms/Product/index.tsx [hunk 2]
```diff
@@ -48,7 +48,7 @@ function Product({ product ,modalSetToggle,className, openQuickView,showQuickVie
           className={`py-[0.625rem] px-6
           w-full text-appTheme border-appTheme border-2 text-[1rem] font-bold  font-HeroNewBold  leading-6 tracking-[0.3px]
           group-hover:bg-appBlackTheme group-hover:text-white group-hover:underline group-hover:border-white
-          hover:bg-appBlackTheme hover:text-white hover:underline hover:border-white ${className}
+          hover:bg-appBlackTheme hover:text-white hover:underline hover:border-white ${className} dark:group-hover:text-black
```
Comment (Vishnu9901) [inline src/libs/ui/src/organisms/Product/index.tsx]: custom classes should always last

File: src/libs/ui/src/templates/ProductListPageTemplate/index.tsx [hunk 3]
```diff
@@ -133,6 +142,16 @@ const PlpPageTemplate: React.FC = () => {
     }
   }, [location.search, setFilters, setSelectedProductCategory]);
 
+  //USed to fetch the products from api call but now  we are fetching from test page
+  const openQuickReviewModal = async (id: number) => {
+    console.log('product id',id);
+    const product = sortedProducts.find(p => p.id === id);
+    console.log('product id',product);
+    if (product) {
```
Comment (Vishnu9901) [inline src/libs/ui/src/templates/ProductListPageTemplate/index.tsx:148]: remove consoles

**Review by Vishnu9901 — CHANGES_REQUESTED** [review]

**Comment by Vishnu9901** [comment]
Great @Nishant8209 , did great job

File: src/libs/ui/assets/Arrow-down-black.63216089.svg
Comment (Nishant8209) [reply src/libs/ui/assets/Arrow-down-black.63216089.svg:1]: WE are using in drop down in plp page

**Review by Vishnu9901 — APPROVED** [review]

**Pull request merged**